### ISSUE TITLE
CAPTCHA: Checkbox border fails contrast ratio against its adjacent colour

### ISSUE THREAD

**Issue by sarasuri** [mmiron-HC/WST-RSAMS#166]
**Page:** All (CAPTCHA under Report a problem or mistake on this page)
**URL:** https://recalls-rappels.canada.ca/en
**User Agent:** Firefox (v99)
**Language:** English and French

**Issue Description:** Under "Report a problem or mistake on this page, CAPTCHA", Checkbox border colour doesn’t pass contrast ratio against its adjacent colour.

Border = #c1c1c1
Adjacent colour= #f9f9f9

![image](https://user-images.githubusercontent.com/12256067/162524340-d19d3261-52b3-4261-b4f7-9c1769f2f5ad.png)

**Steps to reproduce:**

1. Go to the above URL
2. Access the Report a problem or mistake on this page component
3. Visual scanning
4. Use Colour Contrast Analyser to verify the contrast ratio

**Expected results:** The border of the checkbox should have a contrast ratio of at least 3:1 against its adjacent colour.

**Comment by echouin** [comment]
I believe that this is now resolved.

**Comment by YuxuanXie01** [comment]
Retested, issue persists. The contrast ratio of the text colour (#c1c1c1) against the background (#f9f9f9) is still 1.7:1, where it should be at least 3:1.
![image](https://github.com/user-attachments/assets/e2bdfb27-3b48-4468-b037-3cbff75ca8cf)

**Comment by zimw7** [comment]
Issue still exists. The checkbox border color (#c1c1c1) contrast ratio against the background (#f9f9f9) is 1.7:1. It should be at least 3:1.
![image](https://github.com/user-attachments/assets/53f1d4f5-fedf-474a-ab4e-6f3408a901a4)

**Comment by 40999221** [comment]
Issue is still exist. Color contrast ratio for checkbox border with background is 1.7:1 against 3:1 of WCAG standards.

![image](https://github.com/user-attachments/assets/887580d7-bba1-4de8-bfa5-49f974413190)

**Comment by mmiron-HC** [comment]
@zimw7  Please copy this ticket over to [ART-2605](https://jill.hc-sc.gc.ca/jira/browse/ART-2605) Put the status of this one to Escalated and please include a note to say that this is a google product limitation. Add product_limitation to the labels as well
